packages/ui/src/components/va-time-input/VaTimeInput.vue

```diff
@@ -40,23 +40,23 @@
           />
           <va-icon
             v-if="$props.leftIcon"
-            aria-label="toggle dropdown"
+            :aria-label="t('toggleDropdown')"
             v-bind="iconProps"
           />
         </template>
 
         <template #icon>
           <va-icon
             v-if="canBeClearedComputed"
-            aria-label="reset time"
             v-bind="{ ...iconProps, ...clearIconProps }"
+            :aria-label="t('resetTime')"
             @click.stop="reset"
             @keydown.enter.stop="reset"
             @keydown.space.stop="reset"
           />
           <va-icon
             v-else-if="!$props.leftIcon"
-            aria-label="toggle dropdown"
+            :aria-label="t('toggleDropdown')"
```

Comment:
I might miss something to translate. Just run thought :aria-label and props. @aluarius, do you think something else must be translated too?

Reply:
Some aria attributes are set via computed attributes variable. Additional check for 'aria-label' & ariaLabel patterns is needed. Will try to find time next week for this.

All in all, aria attributes and default props values translations is enough probably (can't remember any other cases).